setup.py

```diff
@@ -30,6 +30,7 @@
     "pytz>=2018.9",
     "h5py>=2.9.0",
     "pytimber>=2.8.0",
+    "uncertainties>=3.1.4",
```

Comment:
To be discussed xD

Reply:
(also: are you choosing this version, because it is pi?)

Reply:
> 
> 
> (also: are you choosing this version, because it is pi?)

(nop is the latest, but my fingers are itching to add some comment on the version number)

Reply:
> To be discussed xD

We mostly do our calculations on numpy.arrays or pandas.DataFrame and uncertainties does not seem to be well developed on this front. The performance in terms of CPU time and memory use seems to be orders of magnitude behind that of numpy. The compatibility at least with pandas seems to be questionable. This looks enough not to depend on uncertainties package.

Reply:
I agree with the performance problems and with the pandas problems (which seem to be coming form pandas side). Basic numpy operations work fine though and as we do a lot of `.to_numpy()` anyway this should not be a problem.
I would also not recommend to replace all of our calculations with this uncertainty package, but I see some advantages using it for small scripts like this (where it doesn't matter if the calculation for a 100-number array takes 4us or 400us).

- You can keep errors and values together without mapping them to dictionaries (reduction of variables)
- simple calculations can be performed easily and with errors, without doing the error propagation manually
- significant digits are automatically taken care of by the `__str__()` function

which is basically all this package aims at and it delivers. 
Sure, it would be one more dependency of omc3, but one that I can see be very useful for smaller (i.e. low array length, low repetivity) calculations in the future. 

In case this package is vetoed, I suggest writing a small class that 
- has `.value` and `.error` attributes
- supports implements error propagation for `__add__`, `__sub__`, `__mul__` and `__div__`
- implements `__repr__` as full float and `__str__` as significant digits (like UFloat)
- has a function `round()` which uses the significant digits `float=True` version to round the numbers

Reply:
Also: 
- errors of in-between-steps are also easily visible e.g. for debug.

Not limited to debug, though: If we want to get rid of the uncertainty package, we will need to calculated the errors for the effective betas also manually...

Reply:
Good points.

I still think this has this place in `PyLHC` rather than `omc3` as we don't need another dependency for the package, especially if only used in small scripts.